app/Pods/Fragaria/SMLTextView.m

```diff
@@ -1267,6 +1267,15 @@ - (NSRange)rangeForUserCompletion
     {
         return NSMakeRange(NSNotFound, 0);
     }
+    
```

Comment:
I was hoping to make some progress here, however, this has run into some trouble.
There seem to be two options:
- swizzle the implementation of the methods that I've adjusted
- subclass `SMLTextView` with the changes

I thought subclassing would be preferable to swizzling. In the end I've discovered that I can't easily drop in a subclass for `SMLTextView` and that swizzling is already used (`SMLTextView+CocoaPods.m`).

I'll have another look at swizzling later.